ci-operator/step-registry/osd/gather-extra/osd-gather-extra-ref.metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+	"path": "osd/gather-extra/osd-gather-extra-ref.yaml",
+	"owners": {
+		"approvers": [
+			"jbpratt"
+		],
+		"reviewers": [
+			"jbpratt"
+		]
+	}
+}
```

Comment:
Newline missed.

Reply:
This file is auto generated with `make registry-metadata`. Looking at other metadata files, they do not have a trailing newline.